ios/SiroReactNativeModule.swift

```diff
@@ -24,7 +24,15 @@ public class SiroReactNativeModule: Module {
 
 
     Function("setup") { (environment: String) in
-      SiroSDK.setup(environment: .staging)
+    
+        
+    if let environmentEnum = SKEnvironment(rawValue: environment) {
```

Comment:
I'd prefer to use a guard statement here instead. I'll approve and let you decide.